routes/decoratorQueryArgs.go

```diff
@@ -26,14 +26,16 @@ const (
 	maxUint = ^uint(0)
 	maxInt  = int(maxUint >> 1)
 	minInt  = -maxInt - 1
+	// TagRequiredQueryArg is the tag used to document required query args.
+	TagRequiredQueryArg = "required:allof:queryarg"
 )
 
 type (
 	// QueryArgInt denotes an int query argument. It fulfills the QueryParser
 	// interface.
 	QueryArgInt struct{}
 
-	// QueryArgUint denotes an uint query argument. It fulfills the QueryParser
+	// QueryArgUint denotes a uint query argument. It fulfills the QueryParser
 	// interface.
```

Comment:
It depends on the pronunciation : you would say “an umbrella”, because it starts with [ʌ], while you would say “a union” because it starts with [eə]. I, personnaly, pronounce “uint” [eəInt], hence the “a”.